unit_test.sh

```diff
@@ -20,13 +14,21 @@ IMAGE_TAG=$(git rev-parse --short=7 HEAD)
 DATABASE_USER="compliance"
 DATABASE_PASSWORD="changeme"
 DATABASE_NAME="compliance-test"
-DATABASE_HOST="compliance-db"
+
+function teardown_docker {
+  docker rm -f "$DB_CONTAINER_ID" || true
+  docker rm -f "$TEST_CONTAINER_ID" || true
```

Comment:
@vkrizan yep, you're right - I messed up the variable here. It should be `DB_CONTAINER_NAME` instead of the ID